trace-sdk/src/test/java/io/bitrise/trace/data/management/formatter/BaseDataFormatter.java

```diff
@@ -10,7 +10,7 @@
  * This ensures the ApplicationSessionManager is started before which is required to be able to
  * format any data
  */
-public class BaseDataFormatterTest {
+public class BaseDataFormatter {
```

Comment:
I see your point, but I think this way it is much more confusing, because you would expect some function in it, based on it, but this is for having the common code for test classes. I would keep the original name, and instead make the class abstract, what do you think?

Reply:
Ooh that's a good suggestion. Will update the code and check to make sure Sonar is happy with it too.